apps/lwipserver2/CMakeLists.txt

```
@@ -0,0 +1,50 @@
+#
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+project(lwipserver_single C)
+includeGlobalComponents()
+
+find_package(projects_libs REQUIRED)
+
+set(CAmkESCPP ON CACHE BOOL "" FORCE)
+if("${KernelArch}" STREQUAL "x86")
```

can't we just use the boolean flags
```suggestion
if(KernelArchARM)
...
elseif(KernelArchX86)
...
```